src/test-utils/test-helpers.ts

```diff
@@ -160,8 +162,10 @@ export async function standByForPoxCycle(
   do {
     await standByUntilBurnBlock(lastPoxInfo.current_burnchain_block_height! + 1, api, client);
     lastPoxInfo = await client.getPox();
-  } while (lastPoxInfo.current_cycle.id <= firstPoxInfo.current_cycle.id);
-  expect(lastPoxInfo.current_cycle.id).toBe(firstPoxInfo.next_cycle.id);
```

Comment:
Not sure about this change. Basically `current_cycle.id` can be incorrect and shouldn't really be tested against. I'm thinking most tests rely on the rough behavior here and not the exact value.